### PR TITLE
List mentors traits and allow users to filter mentors by traits

### PR DESCRIPTION
**Pull request by jgradim**
Note: Depends on https://github.com/lifeonmarspt/mentorados-backend/pull/5

### CODE REVIEW

File: app/components/elements/Filters.jsx [hunk 1]
```diff
@@ -17,53 +16,69 @@ class Filters extends React.Component {
...
+          {this.context.meta.traits.map(trait => (
+            <div className="filter" key={trait.id}>
+              <label htmlFor={`filter-trait-${trait.id}`} className="pure-checkbox">
+                {trait.description}
+              </label>
+              <input
+                id={`filter-trait-${trait.id}`}
+                type="checkbox"
+                value={trait.id}
+                checked={(traits || []).find(id => id === trait.id) ? true : false}
```
Comment (hugopeixoto) [inline app/components/elements/Filters.jsx]: We're initializing `this.props.filters.traits` to `[]`, if I'm not mistaken, so we may not need the `|| []`.

File: app/components/elements/Filters.jsx [hunk 2]
```diff
@@ -17,53 +16,69 @@ class Filters extends React.Component {
...
+                {career.description}
+              </label>
               <input
-                id={"filter-career-" + career.id}
+                id={`filter-career-${career.id}`}
                 type="checkbox"
                 value={career.id}
-                checked={(this.props.filters.careers ||  []).find((id) => id === career.id) ? true : false}
-                onChange={(e) => this.handleCareerClick(career, e.target.checked)}
+                checked={(careers || []).find(id => id === career.id) ? true : false}
```
Comment (hugopeixoto) [inline app/components/elements/Filters.jsx]: Same as above. We're initializing `this.props.filters.careers` to `[]`, if I'm not mistaken, so we may not need the `|| []`.

**Review by hugopeixoto — COMMENTED** [review]
I left two comments regarding superfluous code. Feel free to merge this after fixing those, if they're fixable.

File: app/components/elements/Filters.jsx (hunk 1 above)
Comment (jgradim) [reply app/components/elements/Filters.jsx]: Nop, the `|| []` is indeed not necessary.

**Pull request merged**